### DISCUSSION TITLE
Galaxy Brain not working for me

### DISCUSSION THREAD

**Discussion by aliencaocao** [Schweinepriester/github-profile-achievements#535] [Q&A]
I have answered more than 2 discussions:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/5540#discussioncomment-4348519
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/5581#discussioncomment-4364703
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4325#discussioncomment-4063875

Yet the badge is not appearing...

**Comment by SaileshBK** [comment]
I think maybe GitHub is taking your contribution as private.

**Reply by aliencaocao** [reply]
what do you mean? The answered discussion can be seen on my profile publicly

**Reply by SaileshBK** [reply]
hmm, I thought it was not upvoted and I tried upvoting all of your answers. I would say the whole achievement in GitHub is still in beta maybe a weird bug or something where the time you were answering those was down. Try answering in most newly opened discussion?

**Comment by tom-mcIvor** [comment]
please just mark my comment as answered and youll know how to get the galaxy brain

**Reply by aliencaocao** [reply]
> please just mark my comment as answered and youll know how to get the galaxy brain

Stop spamming and providing absolutely useless info

**Reply by SaileshBK** [reply]
lol I agree

**Comment by cybersaksham** [comment]
> I have answered more than 2 discussions: [AUTOMATIC1111/stable-diffusion-webui#5540 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/5540#discussioncomment-4348519) [AUTOMATIC1111/stable-diffusion-webui#5581 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/5581#discussioncomment-4364703) [AUTOMATIC1111/stable-diffusion-webui#4325 (comment)](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4325#discussioncomment-4063875)
> 
> Yet the badge is not appearing...

Congrats, you got it!!! :smile_cat:

**Reply by aliencaocao** [reply]
yea, I had to email github support, this is not something that should be happening...

**Comment by port19x** [comment]
Had the same bug. Found an obscure solution https://github.com/community/community/discussions/43854
